Update from v0.4.1 to v0.5.0 fails

Homebridge v1.10.0, Homebridge UI v4.78.1, Node.js v22.15.0, Raspbian 11 on Raspberry Pi 4 Model B Rev 1.2

```
USER: homebridge
DIR: /usr
CMD: sudo -E -n npm install -g --omit=dev homebridge-mhacwifi1-lan@0.5.0


> node@22.17.0 preinstall
> node installArchSpecificPackage

npm error code E404
npm error 404 Not Found - GET https://registry.npmjs.org/node-linux-arm - Not found
npm error 404
npm error 404  'node-linux-arm@22.17.0' is not in this registry.
npm error 404
npm error 404 Note that you can also install from a
npm error 404 tarball, folder, http url, or git url.
npm error A complete log of this run can be found in: /home/homebridge/.npm/_logs/2025-07-06T12_26_16_733Z-debug-0.log
node:internal/modules/cjs/loader:1404
  throw err;
  ^

Error: Cannot find module 'node-linux-arm/package.json'
Require stack:
- /usr/lib/node_modules/homebridge-mhacwifi1-lan/node_modules/node/installArchSpecificPackage.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1401:15)
    at Function.resolve (node:internal/modules/helpers:145:19)
    at ChildProcess.<anonymous> (/usr/lib/node_modules/homebridge-mhacwifi1-lan/node_modules/node-bin-setup/index.js:20:27)
    at ChildProcess.emit (node:events:518:28)
    at maybeClose (node:internal/child_process:1101:16)
    at ChildProcess._handle.onexit (node:internal/child_process:304:5) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/lib/node_modules/homebridge-mhacwifi1-lan/node_modules/node/installArchSpecificPackage.js'
  ]
}

Node.js v22.15.0
npm error code 1
npm error path /usr/lib/node_modules/homebridge-mhacwifi1-lan/node_modules/node
npm error command failed
npm error command sh -c node installArchSpecificPackage
npm error A complete log of this run can be found in: /home/homebridge/.npm/_logs/2025-07-06T12_26_13_285Z-debug-0.log

Error: Operation failed with code 1.
You can download this log file for future reference.
See https://github.com/homebridge/homebridge-config-ui-x/wiki/Troubleshooting for help.
```

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "homebridge-mhacwifi1-lan",
   "description": "Homebridge plugin for LAN of Mitsubishi Heavy Aircons",
-  "version": "0.5.0",
+  "version": "0.5.1",
   "license": "Apache-2.0",
   "homepage": "https://github.com/rchrch/homebridge-mhacwifi1-lan",
   "repository": {
@@ -33,8 +33,7 @@
     "intesis"
   ],
   "dependencies": {
-    "async-mutex": "^0.5.0",
-    "node": "^22"
+    "async-mutex": "^0.5.0"
   },
   "devDependencies": {
     "@types/debug": "^4.1.12",

